annotation-file-utilities/tests/issue155/Issue155.goal

```diff
@@ -0,0 +1,11 @@
+package test;
+import qual.Q4;
+import qual.Q1;
+import qual.Q2;
```

Comment:
Note that the import for Q3 is missing. This is a separate issue from #155 and I'm trying to minimize it.